app/src/main/res/values-in/strings.xml

```diff
@@ -12,34 +12,36 @@
     <string name="safetynet">SafetyNet</string>
 
     <!--Home-->
-    <string name="no_connection">Koneksi tidak tersedia</string>
+    <string name="no_connection">Tidak tersedia koneksi internet</string>
     <string name="app_changelog">Catatan perubahan</string>
     <string name="loading">Memuat…</string>
     <string name="update">Update</string>
     <string name="not_available">N/A</string>
     <string name="hide">Tutup</string>
-    <string name="home_package">Paket</string>
+    <string name="home_package">ID Paket</string>
+    <string name="home_app_title">Aplikasi</string>
 
+    <string name="home_notice_content">Selalu download Magisk HANYA dari halaman resmi GitHub. File dari sumber yang tidak dikenal bisa jadi berbahaya!</string>
     <string name="home_support_title">Dukung kami</string>
     <string name="home_item_source">Sumber</string>
-    <string name="home_support_content">Magisk gratis dan bersumber terbuka, dan akan selalu seperti itu. Bagaimanapun juga Anda dapat menunjukan kepedulian Anda kepada kami dengan mengirimkan sedikit donasi.</string>
+    <string name="home_support_content">Magisk memang bebas dan open-source, dan akan selalu seperti itu. Namun Anda dapat menunjukan kepedulian Anda kepada kami dengan mengirimkan sedikit donasi.</string>
     <string name="home_installed_version">Terinstal</string>
     <string name="home_latest_version">Terbaru</string>
     <string name="invalid_update_channel">Kanal update tidak valid</string>
     <string name="uninstall_magisk_title">Uninstal Magisk</string>
-    <string name="uninstall_magisk_msg">Semua modul akan dinonaktifkan/dihapus!\nRoot akan dihapus!\nData Anda berpotensi terenkripsi jika belum!</string>
+    <string name="uninstall_magisk_msg">Semua modul akan dinonaktifkan/dihapus!\nAkses root akan dihilangkan!\nData Anda berpotensi terenkripsi jika belum!</string>
```

Comment:
While I'm not opposing this change, original string still says "Root will be removed!"